tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -8,23 +8,34 @@
 """
 import pytest
 
-from ..models.utils import check_outputs_equal
+from ..models.utils import check_logprobs_close, check_outputs_equal
 
-MODELS = [
+EXACT_MATCH_MODELS = [
```

Comment:
Not necessary to call them exact match models.
```suggestion
MODELS = [
```